### PR TITLE
Fixup the test_vmware_methods.py

### PR DESCRIPTION
**Pull request by mfalesni**
There was some domain mishap and also removed 5.2 support.

{{pytest: cfme/tests/automate/test_vmware_methods.py -v --disablemetaplugins blockers }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit be7c6ba4bfb7c077bac04377af8259b12b33d0e0:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 22ac0899c69fa5b474c214486d1b41214c23e0ab:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/automate/test_vmware_methods.py [hunk 1]
```diff
@@ -120,7 +104,6 @@ def test_vmware_vimapi_hotadd_disk(
         request.addfinalizer(lambda: instance.delete() if instance.exists() else None)
         instance.create()
     # Dialog to put the disk capacity
-    return
```
Comment (psav) [inline cfme/tests/automate/test_vmware_methods.py]: What was this doing here?

Comment (mfalesni) [inline cfme/tests/automate/test_vmware_methods.py]: No idea why

**Comment by dajoRH** [comment]
Lint report for commit 3db025855a532d1c3bb1cd946263f318e46142a7:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f1509a8bed4c291f4517c6395362fe102e640804:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0d6e8f39a1fbbd2e4f9dfa92bb45acc550b25021:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cbc4def8419faaf967f09abeaaf03da009664fc6:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
So, this had a run with disabled blockers and failed as expected. If happy, then this can be merged.

**Pull request merged**